db/sql/deploy/buildings/functions/buildings_update_suburb_locality.sql

```diff
@@ -0,0 +1,141 @@
+-- Deploy nz-buildings:buildings/functions/update_suburb_locality_changed_deleted_buildings to pg
+
+BEGIN;
+
+-- building_outlines_update_changed_and_deleted_suburb (replace suburb values with the intersection result)
```

Comment:
Maybe add a comment to clarify that deleted localities aren't processed separately, but rather replaced by added and changed localities.